component-samples/demo/manufacturer/README.md

```diff
@@ -157,12 +157,63 @@ By default, the PRI-Manufacturer uses HTTP for all communications on port 8039.
     Avoid using the default keystore available for production deployment.
 
 # Rendezvous Info
-Commonly referred as RvInfo, is one of the most important configuration of FDO. RvInfo is specified in `MT_SETTINGS` table in the manufacturer storage. It is consumed by device for performing TO1 and by owner through the ownership voucher for performing TO0. Default RvInfo value is: `http://localhost:8040?ipaddress=127.0.0.1&ownerport=8443`
+Commonly referred as RvInfo, is one of the most important configuration of FDO. RvInfo is specified in `MT_SETTINGS` table in the manufacturer storage. It is consumed by device for performing TO1 and by owner through the ownership voucher for performing TO0. Default RvInfo value is: `81858205696c6f63616c686f73748203191f68820c018202447f00000182041920fb` which points to localhost over port 8443 for Owner during TO0 and localhost over port 8040 for device during TO1.
+
```

Comment:
Instead of the bytes representation, put a note here that this is the CBOR representation of `[[[5, "localhost"], [3, 8040], [12, 1], [2, h'7F000001'], [4, 8443]]]`